model/qti/templates/qti.item.tpl.php

```diff
@@ -23,7 +23,9 @@
 <assessmentItem 
     xmlns="http://www.imsglobal.org/xsd/imsqti_v2p1"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    <?php foreach(get_data('namespaces') as $name => $uri):?>xmlns:<?=$name?>="<?=$uri?>"<?php endforeach?>
+    <?php foreach(get_data('namespaces') as $name => $uri):?> 
```

Comment:
This loop generated invalid namespace attributes (without spaces and line breaks):
`xmlns:a="..."xmlns:b=".."`